components/scream/src/share/io/scorpio_input.cpp

```diff
@@ -17,6 +17,10 @@ void AtmosphereInput::pull_input(const std::string& filename, const std::string&
   using namespace scream;
   using namespace scream::scorpio;
 
+  
+  auto gids_dev = m_gm->get_grid(m_grid_name)->get_dofs_gids();
```

Comment:
You already do this during the call to `init`, so you don't need to pull the dof gids again, no?

Reply:
This is a special case where `init` is not called because you want to do everything in one place to pull input without the field manager.  Long term, it might be good to simply get rid of any dependence on the field manager, although this would put the burden on the AD to make sure it iterates over all the important variables in the field manager for things like restarts.  Something we can discuss and then consider for a subsequent PR.